scripts/release/utils.injector.js

```diff
@@ -0,0 +1,277 @@
+#!/usr/bin/env node
+import json2md from 'json2md';
+import fs from 'node:fs';
+import path from 'node:path';
+
+import { Build } from '../../documents/scripts/paths.js';
+import { error, errorHandler, success } from '../helpers/index.js';
+import { ELEMENT_DIST, ELEMENT_SOURCE, generateDocList } from './util.js';
+
+/**
+ * Return remove and replace unwanted text for json2md.
+ * @param {string} text input text
+ * @returns {string} trimmed text
+ */
+const json2mdTrim = (text) => {
+  // replace `. json2md doesn't support `.
+  return text
+    ?.replaceAll('`', "'")
+    .split('\n')
+    .map((e) => e.trim())
+    .join(' ');
+};
+
+/**
+ * Return full text comment from Signature Typedoc structure.
+ * @param {signature} signature typedoc's Signature
+ * @returns {string} text comment
+ */
+const getComment = (signature) => {
+  const summary = signature?.comment?.summary;
+  if (summary && summary.length <= 0) {
```

Comment:
```suggestion
  if (summary?.length <= 0) {
```

Reply:
Don't agree with this. Result of `undefinded <= 0` is false. So I improve this condition to `!summary || summary.length <= 0`.